DEPLOY_LINUX.md

```diff
@@ -0,0 +1,87 @@
+
+## Deploy the Console on Linux
+
+The console can be bound to the same web listener as the controller's management API. The console is then accessed by navigating to the controller's web address.
```

Comment:
> The console can be bound to the same web listener as the controller's management API.

a.) it can be bound to ANY port  the mgmt api is irrelevant
b.) it's too much detail in the wrong place

This should simply be something like:

"With version 3.0 of the Ziti Administration Console and v0.33.0+ of the Ziti Controller, the ZAC can be hosted by the controller. To do this, download the version of ZAC you wish to deploy and configure the controller to host the application:
"
then enumerate the steps...